src/controllers/words.js

```diff
@@ -118,12 +115,13 @@ export const getWord = async (req, res, next) => {
       limit: 1,
       dialects,
       examples,
-    }).then(async ([word]) => {
-      if (!word) {
-        throw new Error('No word exists with the provided id.');
-      }
-      return word;
-    });
+    })
```

Comment:
Again, you'll need to revert this block of code since it's just a formatting change